test_runner/regress/test_gc_old_layers.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import NeonEnvBuilder
+
+
+# Test gc_old_layers
+#
+# This test sets fail point at the end of GC, and checks that pageserver
+# normally restarts after it. Also, there should be GC ERRORs in the log,
+# but the fixture checks the log for any unexpected ERRORs after every
+# test anyway, so it doesn't need any special attention here.
+@pytest.mark.timeout(10000)
+def test_gc_old_layers(neon_env_builder: NeonEnvBuilder):
+    env = neon_env_builder.init_start()
```

Comment:
```suggestion
def test_gc_old_layers(neon_env_builder: NeonEnvBuilder):
    """
    Could we have some comment here how this test is supposed to create "stairs" alike
    draw_timeline_dir output? I cannot see how this ends up creating such, but I understand
    it was the aim.
    """
    env = neon_env_builder.init_start()
```

(placeholder suggestion for a comment, this comment will be reproduced in the allure report)

Reply:
Done

Reply:
The pythonic way to fn comments is still the `"""` string within a function, this way the parser will know to attach it to ast and have it available at runtime. Would still recommend going that way like my suggestion showed.

For example:

```python,
# some fn comment
def foo():
  pass
```

is better as:

```python,
def foo():
  """some fn comment"""
  pass
```

https://peps.python.org/pep-0257/ has more examples.

Reply:
The suggestion seems applied, resolving.